init_processes.sh

```diff
@@ -1,20 +1,31 @@
 #!/bin/bash
 
+if [ -f ".env" ]; then
+    source .env
```

Comment:
`source` is not available in docker container for some reason, throwing `source not found` error. Pushing a fix then approving.

Reply:
Fixed, working fine now.